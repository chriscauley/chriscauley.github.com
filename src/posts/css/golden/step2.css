/* Position each box relative to it's parent */
#golden2 div > div {
  left: 100%;
  position: absolute;
  top: 0;

  /* pivot rotation around top right corner */
  -webkit-transform: rotate(0deg);
  -webkit-transform-origin: top left;
  -webkit-transition: -webkit-transform ease-in 1s;
  transform: rotate(0deg);
  transform-origin: top left;
  transition: transform ease-in 1s;
}

/* The hover stuff */
#golden2 div:hover {
  background: #FFc;
}

#golden2 div:hover > div {
  -webkit-transform: rotate(90deg);
  transform: rotate(90deg);
}

/* hangover from the last step */
#golden2 { height: 100%; overflow: hidden; width: 100%; }
#golden2 > div {
  height: 191px;
  left: 0;
  position: relative;
  width: 309px;
}

#golden2 div {
  background: #999;
  outline: 1px black solid;
}

#golden2 div > div {
  height: 61.8%;
  width: 61.8%;
  z-index: 100;
}
