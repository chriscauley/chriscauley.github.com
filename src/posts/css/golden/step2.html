<link href="./step2.css" rel="stylesheet" />
<div id="golden2">
  <div>
    <div>
      <div>
        <div>
          <div>
            <div>
              <div>
                <div>
                  <div>
                    <div>
                      <div>
                        <div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
